docs/decisions/0004-axios-caching-implementation.rst

```diff
@@ -0,0 +1,87 @@
+Axios Caching Layer Implementation
+=================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The titans team noticed that on a lot of pages the same API calls are repeatedly being made as the user navigates around the website. The data being returned from these requests doesn't change between page loads leading to the same endpoints reloading the same data repeatedly. We decided that we wanted to avoid repeatedly reloading the data and reuse the data that was in the response from the initial load.
```

Comment:
nit: "repeatedly reloading" is kind of redundant, maybe use the phrase "repeatedly loading unchanged data" to represent the thing we want to avoid doing.